front/app/root.tsx

```diff
@@ -60,7 +70,33 @@ function Routes() {
   return useSentryRoutes(routes);
 }
 
+const prefetchData = () => {
```

Comment:
Can we take out of the root component and import the `prefetchData` function? It's nice to keep this file relatively clean (contexts only).